unitTest/taskManager.test.js

```diff
@@ -94,7 +98,51 @@ describe('TaskManager', () => {
       taskManager.getIndex(1);
       taskManager.deleteTask();
 
-      expect(document.querySelectorAll('.task')).toHaveLength(4);
+      expect(document.querySelectorAll('.task')).toHaveLength(6);
+    });
+  });
+
+  describe('Test part 2', () => {
+    test('Status in the task-list is true and false respectively', () => {
+      expect(taskManager.tasks[0].completed).toBe(true);
+      expect(taskManager.tasks[1].completed).toBe(false);
+    });
+
+    test('Contents in task-list are "Task 1" and "Task 2 respectively"', () => {
+      expect(taskManager.tasks[0].description).toBe('Task 1');
+      expect(taskManager.tasks[1].description).toBe('Task 2');
+    });
+
+    test('Tasks re-postion"', () => {
+      let lastChild = taskManager.tasks.length - 1;
+
+      expect(taskManager.tasks[lastChild].index).toBe(4);
+      // Deleting "Task 3"
+      taskManager.getIndex(2);
+      taskManager.deleteTask();
```

Comment:
- [ ] Well done for the tests you have written so far 👏. Please write tests that cover the following functions:
1. the function for editing the task description
2. the "Clear completed" function in your application 👍